package.json

```diff
@@ -8,12 +8,12 @@
     "build": "cpx index.d.ts dist/ && tsc -p .",
     "build:watch": "cpx index.d.ts dist/ && tsc -p . -w",
     "clean": "rimraf dist/**/* && rimraf tests/simple/coverage && rimraf tests/simple-async/coverage && rimraf tests/**/*/debug.txt",
-    "clean-build": "npm run clean && npm run build",
-    "pretest": "npm run tslint && npm run clean-build",
+    "clean-build": "yarn clean && yarn build",
```

Comment:
my concern with this is that this won't work for folks who don't have yarn installed.

Reply:
You are right about that. What I thought was since we use yarn.lock to maintain depedencies consistency, the forks should use yarn too. But if you wish to use npm, I'll revert this file

Reply:
The other part of the PR (doc) is nice.

If you could also update the AUTHORS file with your name in alphabetic order, I'll merge this in right away